node/node.go

```diff
@@ -870,7 +870,7 @@ func New(
 	consensusObj *consensus.Consensus,
 	chainDBFactory shardchain.DBFactory,
 	blacklist map[common.Address]struct{},
-	isArchival bool,
```

Comment:
suggest rename isArchival (usually a bool) to archivalModes or archivalSettings etc.